lib/krb5/fcache.c

```diff
@@ -393,7 +393,7 @@ fcc_open(krb5_context context,
     size_t tries = 3;
 #endif
     int strict_checking;
-    int fd;
+    int fd = -1;
```

Comment:
Unnecessary as there is no use of 'fd' before assignment on line 425.   Assuming it was necessary the assignment would have to go after the again: label on line 410.